ERROR - 2025-02-21 17:16:07 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u372271345_petz'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php(185): mysqli->real_connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\Petz24\app\Controllers\BookingController.php(16): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\BookingController->getPetList()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#6 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\Petz24\app\Controllers\BookingController.php(16): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\BookingController->getPetList()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#6 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2025-02-21 17:16:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\BookingController.php(16): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->getPetList()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-02-21 17:16:07 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u372271345_petz'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php(185): mysqli->real_connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\Petz24\app\Controllers\UserPetController.php(13): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\UserPetController->getUserPetList()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#6 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\Petz24\app\Controllers\UserPetController.php(13): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\UserPetController->getUserPetList()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#6 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2025-02-21 17:16:07 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserPetController.php(13): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UserPetController->getUserPetList()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-02-21 17:16:17 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u372271345_petz'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php(185): mysqli->real_connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\Petz24\app\Controllers\UserPetController.php(13): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\UserPetController->getUserPetList()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#6 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\Petz24\app\Controllers\UserPetController.php(13): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\UserPetController->getUserPetList()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#6 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2025-02-21 17:16:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserPetController.php(13): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UserPetController->getUserPetList()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-02-21 17:16:17 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u372271345_petz'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php(185): mysqli->real_connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\Petz24\app\Controllers\BookingController.php(16): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\BookingController->getPetList()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#6 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\Petz24\app\Controllers\BookingController.php(16): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\BookingController->getPetList()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#6 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2025-02-21 17:16:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\BookingController.php(16): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->getPetList()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-02-21 17:16:38 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u372271345_petz'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php(185): mysqli->real_connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\Petz24\app\Controllers\BookingController.php(16): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\BookingController->getPetList()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#6 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\Petz24\app\Controllers\BookingController.php(16): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\BookingController->getPetList()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#6 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2025-02-21 17:16:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\BookingController.php(16): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->getPetList()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
ERROR - 2025-02-21 17:16:38 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'u372271345_petz'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php(185): mysqli->real_connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#2 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\projects\Petz24\app\Controllers\UserPetController.php(13): CodeIgniter\Database\BaseConnection->query()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\UserPetController->getUserPetList()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#6 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\projects\Petz24\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(392): CodeIgniter\Database\MySQLi\Connection->connect()
#1 C:\xampp\htdocs\projects\Petz24\system\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\projects\Petz24\app\Controllers\UserPetController.php(13): CodeIgniter\Database\BaseConnection->query()
#3 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(941): App\Controllers\UserPetController->getUserPetList()
#4 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
#5 C:\xampp\htdocs\projects\Petz24\system\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
#6 C:\xampp\htdocs\projects\Petz24\index.php(79): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2025-02-21 17:16:38 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 428.
 1 SYSTEMPATH\Database\BaseConnection.php(575): CodeIgniter\Database\BaseConnection->initialize()
 2 APPPATH\Controllers\UserPetController.php(13): CodeIgniter\Database\BaseConnection->query()
 3 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\UserPetController->getUserPetList()
 4 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 5 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 6 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:19:05 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 88.
 1 APPPATH\Controllers\BookingController.php(88): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:19:07 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 88.
 1 APPPATH\Controllers\BookingController.php(88): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:19:07 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 88.
 1 APPPATH\Controllers\BookingController.php(88): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:19:08 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 88.
 1 APPPATH\Controllers\BookingController.php(88): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:19:13 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 88.
 1 APPPATH\Controllers\BookingController.php(88): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:21:11 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 88.
 1 APPPATH\Controllers\BookingController.php(88): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:21:22 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 88.
 1 APPPATH\Controllers\BookingController.php(88): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:22:01 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 88.
 1 APPPATH\Controllers\BookingController.php(88): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:24:50 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 90.
 1 APPPATH\Controllers\BookingController.php(90): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:25:12 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 90.
 1 APPPATH\Controllers\BookingController.php(90): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:27:48 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 95.
 1 APPPATH\Controllers\BookingController.php(95): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:29:12 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 95.
 1 APPPATH\Controllers\BookingController.php(95): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-02-21 17:30:11 --> ErrorException: Undefined array key "address_id"
in APPPATH\Controllers\BookingController.php on line 91.
 1 APPPATH\Controllers\BookingController.php(91): CodeIgniter\Debug\Exceptions->errorHandler()
 2 SYSTEMPATH\CodeIgniter.php(941): App\Controllers\BookingController->addBooking()
 3 SYSTEMPATH\CodeIgniter.php(502): CodeIgniter\CodeIgniter->runController()
 4 SYSTEMPATH\CodeIgniter.php(361): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
